skiko/src/commonMain/kotlin/org/jetbrains/skia/ImageFilter.kt

```diff
@@ -381,6 +381,46 @@ class ImageFilter internal constructor(ptr: NativePointer) : RefCnt(ptr) {
             }
         }
 
+        fun makeRuntimeShader(runtimeShaderBuilder: RuntimeShaderBuilder, shaderName: String, input: ImageFilter?): ImageFilter {
```

Comment:
Could you add tests for these shaders in ImageFilterTest? At least smoke tests (just call methods, to exclude crashes), if it is not trivial.

Reply:
Added in https://github.com/JetBrains/skiko/pull/520/commits/25a6029a5f5f049f911a2487c5e446f02bc3c60d